src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -193,21 +194,30 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
 
   // Converts the Traversable[Call] to Seq[DBIOAction[]] that insert the correct rows
   private def toCallActions(workflowExecution: WorkflowExecution, backend: Backend,
-                            calls: Traversable[Call]): Seq[DBIO[Unit]] = {
-    def toWorkflowExecutionCallAction(call: Call) = toCallAction(workflowExecution, backend, call)
-    calls.toSeq map toWorkflowExecutionCallAction
+                            keys: Traversable[CallKey]): Seq[DBIO[Unit]] = {
+    def toWorkflowExecutionCallAction(key: CallKey) = toCallAction(workflowExecution, backend, key)
+    keys.toSeq map toWorkflowExecutionCallAction
```

Comment:
This is currently the only usage of `toCallAction`, you could do away with the local `toWorkflowExecutionCallAction` function if the parameter list to the former were curried:

```scala
  private def toCallAction(workflowExecution: WorkflowExecution, backend: Backend)(key: CallKey) = ???
```

then

```scala
keys.toSeq map toWorkflowExecutionCallAction(workflowExecution, backend)

```